modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -2,20 +2,28 @@
 :toc: right
 :page-aliases: configuration/server/antivirus_configuration.adoc
 :clamav-url: http://www.clamav.net/index.html
+:icap-url: https://tools.ietf.org/html/rfc3507
+:c-icap-url: https://sourceforge.net/p/c-icap/wiki/ModulesConfiguration/
 
 == Introduction
 
-{clamav-url}[ClamAV] is the only _officially_ supported virus scanner available for use with ownCloud. 
+When sharing files, security is a key aspect. The ownCloud {anti-virus-app-url}[Anti-Virus] extension helps by protecting against malicious software like trojans or viruses. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Webinterface menu:Admin[Settings > Security].
```

Comment:
```suggestion
When sharing files, security is a key aspect. The ownCloud {oc-marketplace-url}/apps/files_antivirus[Anti-Virus] extension helps by protecting against malicious software like trojans or viruses. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Webinterface menu:Admin[Settings > Security].
```